yb-voyager/src/query/queryissue/parser_issue_detector.go

```diff
@@ -323,16 +319,6 @@ func (p *ParserIssueDetector) GetPercentTypeSyntaxIssues(query string) ([]QueryI
 }
 
 func (p *ParserIssueDetector) GetDMLIssues(query string, targetDbVersion *ybversion.YBVersion) ([]QueryIssue, error) {
-	issues, err := p.getDMLIssues(query)
-	if err != nil {
-		return issues, err
-	}
-
-	return p.getIssuesNotFixedInTargetDbVersion(issues, targetDbVersion)
-}
-
-// TODO: in future when we will DDL issues detection here we need `GetDDLIssues`
-func (p *ParserIssueDetector) getDMLIssues(query string) ([]QueryIssue, error) {
```

Comment:
would recommend still keeping the `getDMLIssues` and `GetDMLIssues` both. The idea was to not touch the `GetDMLIssues` often and keep the `getIssuesNotFixedInTargetDbVersion` in it so that it does not get missed. 

If we have just one function, there is a high possibility that we accidentally just write some return from the middle of that function which will lead to  issues not being filtered out..